lib/openhab/core/types/hsb_type.rb

```diff
@@ -12,6 +12,11 @@ module Types
       # {HSBType} is a complex type with constituents for hue, saturation and
       #  brightness and can be used for color items.
       class HSBType < PercentType
+        if OpenHAB::Core.version >= OpenHAB::Core::V4_3
```

Comment:
ColorUtil itself was [added in 4.0](https://github.com/openhab/openhab-core/pull/3434)

Reply:
might as well.